core/common/src/main/java/alluxio/wire/MasterWebUIConfiguration.java

```diff
@@ -28,6 +28,10 @@ public final class MasterWebUIConfiguration implements Serializable {
 
   private List<String> mWhitelist;
   private TreeSet<Triple<String, String, String>> mConfiguration;
+  private String mClusterConfigHash;
+  private String mPathConfigHash;
+  private String mClusterConfigLastUpdateTime;
+  private String mPathConfigLastUpdateTime;
```

Comment:
can we make all these four variables one wrapper object?

```
class ConfigHash {
  private final String mHashValue;
  private final long mLastUpdateMs;
}
```

Reply:
done